modules/rosa-sts-oidc-provider-command.adoc

```diff
@@ -21,8 +21,9 @@ When using `manual` mode, the `aws` command is printed to the terminal for your
 [source,terminal]
 ----
 aws iam create-open-id-connect-provider \
-	--url https://rh-oidc.s3.<aws_region>.amazonaws.com/<cluster_id> \
+	--url https://rh-oidc.s3.<aws_region>.amazonaws.com/<cluster_id> \ <1>
 	--client-id-list openshift sts.amazonaws.com \
-	--thumbprint-list <thumbprint> <1>
+	--thumbprint-list <thumbprint> <2>
 ----
+<1> The URL used to create the OpenID Connect (OIDC) identity provider is obscured with a CloudFront URL once the cluster is created.
```

Comment:
The URL used to reach the OpenID Connect (OIDC) identity provider once the cluster is created.